backend/src/test/java/codesquard/app/api/wishitem/WishItemServiceTest.java

```diff
@@ -45,7 +43,7 @@ void wishRegisterTest() {
 		Item item1 = supportRepository.save(request.toEntity(member, "thumbnail"));
 
 		// when
-		wishItemService.register(item1.getId(), member.getId());
+		wishItemService.changeWishStatus(item1.getId(), member.getId(), WishStatus.YES);
```

Comment:
```suggestion
		wishItemService.changeWishStatus(item1.getId(), member.getId(), YES);
```
정적 메소드 임포트를 사용하여 줄일 수 있을 것 같습니다.

Reply:
어떤것에 대한 YES인지를 판별하려면 static import는 하지 않는게 좋을거같다고 생각했습니다